app/controllers/search_controller.rb

```diff
@@ -3,8 +3,8 @@ class SearchController < ApplicationController
   def search
     @name = params[:summoner]
     if valid_search?
```

Comment:
Good catch.  That definitely broke when i refactored the search stuff.  Fixed it here: https://github.com/danfstucky/LeagueSpec/pull/50/commits/cd3db2c0588ff1b1eaf385dc9e5dcc32aa6a8241